operator/option/config.go

```diff
@@ -453,6 +459,12 @@ func (c *OperatorConfig) Populate() {
 		c.IPAMSubnetsTags = m
 	}
 
+	if m, err := command.GetStringMapStringE(viper.GetViper(), IPAMInstancesTags); err != nil {
+		log.Fatalf("unable to parse %s: %s", IPAMInstancesTags, err)
+	} else if len(m) != 0 {
```

Comment:
We had to remove this check, see #19172 for how this now looks on current master
```suggestion
	} else {
```

Reply:
I just rebased the PR on master